models/rubicon/optimism/rubicon_optimism_schema.yml

```
@@ -93,4 +93,112 @@ models:
         description: ""
       - &evt_index
         name: evt_index
-        description: ""
\ No newline at end of file
+        description: "the event index of the trade"
+  - name: rubicon_optimism_offers
+    meta:
+      blockchain: optimism
+      project: rubicon
+      contributors: denver
+    config:
+      tags: ['optimism','rubicon','dex']
+    description: >
+        A table containing all offers of rubicon on optimism
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
```

We can also apply a `check_seed` test here:
```suggestion
    tests:
      - check_seed:
          seed_file: ref('rubicon_optimism_offers_test_data')
          match_columns:
            - block_number
            - tx_hash
            - evt_index
            - tx_index
          check_columns:
            - l1_gas_used
            ...
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
```

---

implemented [here](https://github.com/duneanalytics/spellbook/pull/3042/commits/a364fd2746803871fde9794ca71fed037476393a), ty @Hosuke